app/backend/approaches/AskData.py

```diff
@@ -45,121 +46,146 @@
 # # Access environment variables
 # azure_openai_service_key = os.getenv("AZURE_OPENAI_SERVICE_KEY")
 # azure_openai_service = os.getenv("AZURE_OPENAI_SERVICE")
-# azure_openai_chatgpt_deployment = os.getenv("AZURE_OPENAI_CHATGPT_DEPLOYMENT")
+azure_openai_chatgpt_deployment = os.getenv("AZURE_OPENAI_CHATGPT_DEPLOYMENT")
 
 # openai.api_key = azure_openai_service_key
 # openai.api_base = f"https://{azure_openai_service}.openai.azure.com/"
-# deployment_name = azure_openai_chatgpt_deployment
+deployment_name = azure_openai_chatgpt_deployment
+OPENAI_DEPLOYMENT_NAME = deployment_name
 
 # openai.api_type = "azure"
 # openai.api_version = "2023-06-01-preview"
 # 
 #______________________________________________________________________________________
 
 
-
-
 # Page title
 
 
-#uploaded_file = st.file_uploader('Upload a CSV file', type=['csv'])
+# uploaded_file = st.file_uploader('Upload a CSV file', type=['csv'])
+dffinal = None
+pdagent = None
+agent_imgs = []
 
 
 def save_chart(query):
```

Comment:
Are these chart not going to saved anywhere ? I havent seen this in action so asking here. If they are going to saved then we will need to give a file name here if not we are fine.

Reply:
For the session, the base64 encodings are saved in the agent_imgs list and the get charts function is used to retrieve any charts that have been created.